Source/Redemption/Private/BTTask_DefineNextStandingPoint.cpp

```diff
@@ -92,6 +94,11 @@ APatrolPathIndicator* UBTTask_DefineNextStandingPoint::SecondClosestPositionToPl
 			ClosestDistanceToPlayer = FVector::Dist(PlayerLocation, Current->GetActorLocation());
 			Closest = Current;
 		}
+		else if (FVector::Dist(PlayerLocation, Current->GetActorLocation()) < SecondDistanceToPlayer)
+		{
+			SecondDistanceToPlayer = FVector::Dist(PlayerLocation, Current->GetActorLocation());
+			SecondClosest = Current;
```

Comment:
The reason why you set the second point was because you want to check the next point location after the gargoyle moves?

Reply:
the gargoyle moves to the closest place to the player. But if it is already on the closest it moves to the second closest. this function is to find the second closest point.
this change in code is made so even if the closest variable is already set to the closest it will still update the second closest variable